packages/web-wallet/vitest.config.ts

```diff
@@ -1,9 +1,7 @@
 import { defineConfig } from 'vitest/config';
```

Comment:
Didn't we remove vite, why keep this file?

Reply:
Because we're still using `vitest` for the tests